src/com/company/Main.java

```diff
@@ -36,7 +36,7 @@ public static String readFileAsString(String fileName)
               return data;
     }
 
-        public static void checkforsupport(File file)
+        public static void checkForSupport(File file)
```

Comment:
You should ideally be throwing an exception from here or atleast return false if the file is not in a supported format

Reply:
i have resolved that in a  recently committed code.